api/tests/core/bookings/test_api.py

```diff
@@ -1448,7 +1448,7 @@ def test_returns_two_days_before_event_if_event_begins_between_two_and_four_days
         ) == event_date_four_days_from_now - timedelta(days=2)
 
 
-@freeze_time("2032-11-17 15:00:00")
+@freeze_time()
```

Comment:
On peut enlever le décorateur :D

Reply:
Non, il ne marche pas sans !
En fait, la fonction testée utilise utcnow() aussi, et compare les dates ensuite, donc on est obligé de freeze pour avoir une égalité. Après on pourrait comparer les deux dates en tronquant les heures pour éviter des cas flaky, mais je n'ai pas eu le temps de le faire pendant la journée backend

Reply:
on pourrait pas faire:

```suggestion
        now = datetime.utcnow()
        recent_booking_beginning_datetime = now + timedelta(days=90)
        old_booking_beginning_datetime = now - timedelta(days=7)
        tomorrow = now + timedelta(days=1)
```

?

Reply:
J'ai essayé, mais la fonction testée fait un utcnow(), donc utilise une valeur différente du `now`

Reply:
Quand j'ai testé ça marchait pas :o
Ah, mais tu as mis un db_session en plus... C'est ce qui change tout ?

Reply:
ah non j’ai oublié de l’enlever je faisais autre chose en même temps

Reply:
Je vais rajouter ces modifs alors :)
Merci !